homeassistant/components/teslemetry/sensor.py

```diff
@@ -449,6 +449,11 @@ def __init__(
         """Initialize the sensor."""
         super().__init__(vehicle, description.key)
 
+    @property
+    def native_value(self) -> StateType:
+        """Return the state of the sensor."""
+        return self._value
```

Comment:
By default, [`native_value` in the sensor entity](https://github.com/home-assistant/core/blob/6422bc4c19250397a41d1592cc3339c9568aec7c/homeassistant/components/sensor/__init__.py#L458-L461C21) returns the `_attr_native_value` attribute which could be used instead of overriding the property, but I see that the [`_value` property](https://github.com/home-assistant/core/blob/c1cc946240677c9cf293d57e58a400f01351518e/homeassistant/components/teslemetry/entity.py#L51-L54C21) comes from the coordinator, so this is fine.